### ISSUE TITLE
`@project-serum/borsh` dependency causes `@solana/web3.js` version conflicts

### ISSUE THREAD

**Issue by jstarry** [project-serum/anchor#252]
#### Problem
Adding `@project-serum/anchor` to a project with a `@solana/web3.js` v1.x dependency causes dependency resolution issues due to `@project-serum/borsh`'s peer dependency on `@solana/web3.js` v0.90

```
$ npm i --save @project-serum/anchor
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: @solana/web3.js@1.9.0
npm ERR! node_modules/@solana/web3.js
npm ERR!   @solana/web3.js@"^1.7.0" from the root project
npm ERR!   @solana/web3.js@"^1.2.0" from @project-serum/anchor@0.4.5
npm ERR!   node_modules/@project-serum/anchor
npm ERR!     @project-serum/anchor@"*" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @solana/web3.js@"^0.90.0" from @project-serum/borsh@0.1.1
npm ERR! node_modules/@project-serum/borsh
npm ERR!   @project-serum/borsh@"^0.1.1" from @project-serum/anchor@0.4.5
npm ERR!   node_modules/@project-serum/anchor
npm ERR!     @project-serum/anchor@"*" from the root project
```

I would fix this myself but wasn't sure where to find Serum's fork of borsh-ts

**Comment by jstarry** [comment]
cc @armaniferrante

**Comment by armaniferrante** [comment]
Thanks @jstarry! For future reference, the borsh package is hidden away in the catchall ts lerna repo https://github.com/project-serum/serum-ts/tree/master/packages/borsh. I'll fix this though!

**Comment by armaniferrante** [comment]
Should be addressed by v0.2.1. Please re-open if this is a problem still.
